web/themes/contrib/civictheme/civictheme_library/components/03-organisms/mobile-navigation/mobile-navigation.twig

```diff
@@ -16,7 +16,7 @@
 {% set theme_class = 'ct-theme-%s'|format(theme|default('light')) %}
 {% set modifier_class = '%s %s'|format(theme_class, modifier_class|default('')) %}
 
-<div class="ct-mobile-navigation {{ modifier_class }}" data-flyout data-flyout-direction="right">
+<div class="ct-mobile-navigation {{ modifier_class }}" data-flyout data-flyout-direction="right" data-flyout-focus>
```

Comment:
@alan-cole 
can we change the `data-flyout-focus` to support accepting a comma-separated list of selectors that would be focused on in the order they declared.

This would give the focus control to the consumer component

```
div class="ct-mobile-navigation {{ modifier_class }}" data-flyout data-flyout-direction="right" data-flyout-focus=".ct-link">
```

The final list of focus elements will be:
```
...elements discovered with data-flyout-focus...,
[data-flyout-close-trigger],
[data-flyout-close-all-trigger]
```

this would make sure that close and close-all trigger are always present in the list and, if the list is extended, they are always last in the list.

Reply:
Updated the implementation as per your comment.